content/configuration/stats.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Stats
+contributors:
+  - SpaceK33z
+---
+
+The `stats` option lets you precisely control what bundle information gets displayed. This can be a nice middle ground if you don't want to use `quiet` or `noInfo` because you want some bundle information, but not all of it.
+
+T> For webpack-dev-server, this property needs to be in the `devServer` object.
+
+### `stats`
+
+`object` `string`
+
+There are some presets: `none`, `errors-only`, `minimal` and `verbose`. Use them like this:
+
+```js
+stats: "errors-only"
+```
+
+For more granular control, it is possible to specify exactly what information you want:
+
+```js
+stats: {
+  chunks: false,
+  hash: false
+}
+```
+
+The available options are:
```

Comment:
Aii, I feared that you would ask this. I agree with you, but is it okay if we do that later? These options are not documented anywhere (in fact, all that I've written here comes from reading the source directly), so it costs me a lot of time to find out what everything stands for.

Also note that this wasn't even documented before, so I think for the MVP this is enough. I think it's better to focus on other tickets that need to be ready for the MVP.